src/components/activities.js

```diff
@@ -216,7 +231,10 @@ export class ActivityIframePort {
    * @template T
    */
   on(message, callback) {
-    const label = getLabel(message);
+    let label = null;
+    try {
+      label = getLabel(message);
+    } catch (ex) {}
```

Comment:
Could you add a comment describing why this `catch` block is empty?

Reply:
I understand why it's empty, but if we could make that really obvious for future editors who might be in a rush, that would be awesome :)

Reply:
done